testsuite/features/secondary/min_migrate_to_trad.feature

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# This test case does not test a valid scenario. In SUMA you normally do not
+# delete a traditional client and then register it again as Salt minion. You
+# would always just bootstrap the traditional client as a Salt minion again
+# and Salt would recognize that it is a traditional client and will take the
+# the necessary steps to migrate it
+
+Feature: Migrate a Salt minion into a traditional client
```

Comment:
I am not sure if I should add a scope here. If so I think we should add both, `@sle_client` and `@sle_minion`.

Reply:
We for sure need @sle_client as we make use of sle_client VM.
But this is the tag that assure that this test is not triggered when the VM is not deployed.

Another thing, are the scopes.
Scopes allow us to filter a group of features, to run only features that are on the same scope.
In that case, looking the list, https://github.com/uyuni-project/uyuni/blob/master/testsuite/config/cucumber.yml , we can either create a new scope for @scope_client_migration or if you don't want to make much complex this PR, you can use @scope_traditional_client and that's fine for now. Maybe in another PR, if you want to play with scope, you add the new one. As it's not only here that we need to add, but also into the pipeline definitions in susemanager-ci repo

Reply:
I'm biased here because that's exactly what I did but yes, I'd add `@sle_client` and `@sle_minion` if you need both. But, from looking at your tests, do you _need_ `@sle_minion`? :thinking: you start off from the sle_client right? maybe that'd be enough for the tag here :thinking: I do think you should have a scope though but I'm not sure which of the following would be better: `@scope_salt`, `@scope_traditional_client` or maybe making a new scope like `@scope_client_migration` since it seems like its own specific use case

Reply:
Thanks for the extensive comments. I added two tags/scops now.